src/types/geography.ts

```diff
@@ -0,0 +1,301 @@
+export class Geography {
+  private point: GeographyPoint;
+  private line: GeographyLineString;
+  private polygon: GeographyPolygon;
+  featureType: GeographyFeatureType;
+
+  constructor(str = "") {
+    this.parse(str);
+  }
+
+  /**
+   * Return longitude (horizontal) coordinate for the point
+   * @returns number
+   */
+  get longitude(): number {
+    return this.point?.longitude || 0;
+  }
+
+  /**
+   * Return latitude (vertical) coordinate for the point
+   * @returns number
+   */
+  get latitude(): number {
+    return this.point?.latitude || 0;
+  }
+
+  /**
+   * Return Spatial Reference System Identifier (SRID), identifies de reference system for the represented Geographic object
+   * @returns number
+   */
+  get srid(): number {
+    // TODO: not implemented
+    return 0;
+  }
+
+  /**
+   * Return true if the Geography is emtpy
+   * @returns boolean
+   */
+  isEmpty(): boolean {
+    return this.featureType === "";
+  }
+
+  /**
+   * Return true if the Geography is null
+   * @returns boolean
+   */
+  isNull(): boolean {
+    return this.featureType === "";
+  }
+
+  /**
+   * Set Geography to empty value
+   */
+  setEmpty() {
+    this.point = undefined;
+    this.line = undefined;
+    this.polygon = undefined;
+    this.featureType = "";
+  }
+
+  /**
+   * Serializes the object to string (alias for toWkt)
+   * @returns string
+   */
+  toString(): string {
+    return this.toWkt();
+  }
+
+  /**
+   * Serializes the object to WKT representation
+   * @returns string
+   */
+  toWkt(): string {
+    switch (this.featureType) {
+      case "POINT":
+        return `POINT (${this.longitude} ${this.latitude})`;
+      case "LINE":
+        return `LINESTRING (${this.line
+          .map(point => `${point.longitude} ${point.latitude}`)
+          .join(", ")})`;
+      case "POLYGON":
+        return `POLYGON (${this.polygon
+          .map(
+            ring =>
+              `(${ring
+                .map(point => `${point.longitude} ${point.latitude}`)
+                .join(", ")})`
+          )
+          .join(", ")})`;
+      default:
+        return "";
+    }
+  }
+
+  /**
+   * Serializes the object to GeoJSON representation
+   * @returns string
+   */
+  toGeoJson(): string {
+    switch (this.featureType) {
+      case "POINT":
+        return `{"type":"Point","coordinates":[${this.longitude},${this.latitude}]}`;
+      case "LINE":
+        return `{"type":"LineString","coordinates":[${this.line
+          .map(point => `[${point.longitude},${point.latitude}]`)
+          .join(",")}]}`;
+      case "POLYGON":
+        return `{"type":"Polygon","coordinates":[${this.polygon
+          .map(
+            ring =>
+              `[${ring
+                .map(point => `${point.longitude} ${point.latitude}`)
+                .join(",")}]`
+          )
+          .join(",")}]}`;
+      default:
+        return '{"type":"GeometryCollection","coordinates":[]}';
+    }
```

Comment:
Did you consider creating an object and using `JSON.stringify` to get the JSON string?